indexer/composite_id.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "base/geo_object_id.hpp"
+
+#include <string>
+
+#include <boost/functional/hash.hpp>
+
+namespace indexer
+{
+// This struct represents a composite Id.
+// This will be useful if we want to distinguish between polygons in a multipolygon.
```

Comment:
Please write more comprehensive documentation.
1. What do you mean by "composite"?
2. How does it help to distinguish that?